components/PagePost.vue

```diff
@@ -0,0 +1,88 @@
+<!-- Component props: title, author,date ,image, content -->
+<template>
+  <section class="bg-mau-primary-50 h-full my-2">
+    <div class="container mx-auto">
+      <div class="px-12 md:px-24 lg:px-56 my-5">
+        <!-- Title -->
+        <p
+          class="
+            font-medium
+            md:text-6xl
+            text-5xl text-mau-primary-700
+            my-1
+            md:text-left
+            text-center
+          "
+        >
+          {{ title }}
+        </p>
+        <!-- Author -->
+        <p
+          class="
+            font-light
+            text-md text-mau-primary-700
+            mt-1
+            md:text-left
+            text-center
+          "
+        >
+          By: {{ author }}
+        </p>
+        <!-- Date -->
+        <p
+          class="
+            font-light
+            text-md text-mau-primary-700
+            mt-1
+            md:text-left
+            text-center
+          "
+        >
+          {{ date }}
+        </p>
+        <!-- Image -->
+
+        <img :src="img" class="my-5" />
+
+        <!-- Content -->
+        <p
+          class="
+            font-normal
+            md:text-xl
+            text-lg text-mau-primary-700
+            my-4
+            py-6
+            text-justify
+          "
+        >
+          {{ content }}
```

Comment:
This is the incorrect way to display content because if do it this way, it will be a normal string and no formatting from the markdown at all.

The correct way is to do with the `<nuxt-content :document="post" />` component. See the documentation: https://www.netlifycms.org/docs/nuxt/#example-blog-post